pallets/stellar-bridge/src/lib.rs

```diff
@@ -263,14 +263,14 @@ pub mod pallet {
 			let DepositPayload { currency_id, amount, destination, signed_by } = payload;
 
 			log::info!(
-				"submit_deposit_unsigned_with_signed_payload: ({:?}, {:?}, {:?})",
+				"[OCW] Submit_deposit_unsigned_with_signed_payload: ({:?}, {:?}, {:?})",
 				amount,
 				destination,
 				signed_by
 			);
 
 			let result = T::Currency::deposit(currency_id, &destination, amount);
-			log::info!("{:?}", result);
+			log::info!("[OCW] {:?}", result);
```

Comment:
I was looking into this call to see if we could include a prefix in a nicer way, but we can't. A more elegant solution could be wrap the call in a logger defined here, but also this is out of the scope of this pr.